README.md

```diff
@@ -10,10 +10,34 @@ The Vault Operator deploys and initializes Vault on Kubernetes, and runs in high
 
 Deploy the charm:
 ```bash
-juju deploy vault-k8s -n 5 --trust
+juju deploy vault-k8s --channel edge -n 5 --trust
 ```
 > Note: It is advised to deploy Vault with an odd number of units
 
+### Access Vault through Ingress
+
+Deploy Traefik
+
+```bash
+juju deploy traefik-k8s --channel edge --trust --config external_hostname=<your hostname> 
+```
+
+#### Integrate Traefik with Self-Signed-Certificates Operator
+
+```bash
+juju deploy self-signed-certificates --channel beta
+juju integrate self-signed-certificates:certificates traefik-k8s
```

Comment:
Please also specify the traefik integration name (`certificates` or whatever it is)

Reply:
Done